src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -955,6 +953,7 @@ async customSubmit(value) {
       this.ws.call('vm.create', [vm_payload]).subscribe(vm_res => {
         this.loader.close();
         this.modalService.close('slide-in-form');
+        this.modalService.refreshTable();
```

Comment:
I have a similar issue and was curious if there any objections to call `modalService .refreshTable()` directly in `modalService .close()`?

Reply:
Maybe instead of `refreshTable` we could call it `onClose`? That way it comes a wider array of scenarios and makes sense to use it inside `modalService.close()`.

Reply:
Agreed, makes sense